ui/webui/src/components/installation/InstallationProgress.jsx

```diff
@@ -97,31 +123,40 @@ export class InstallationProgress extends React.Component {
         }
 
         return (
-            <Bullseye className={idPrefix + "-status-" + status}>
-                <EmptyStatePanel
-                  icon={icon}
-                  loading={!icon}
-                  paragraph={
-                      status !== "success"
-                          ? <Progress
-                              id={idPrefix + "-bar"}
-                              label={cockpit.format("$0 of $1", step, steps)}
-                              max={steps}
-                              min={0}
-                              title={message}
-                              value={step}
-                              valueText={cockpit.format("$0 of $1", step, steps)}
-                              variant={status}
-                          />
-                          : <Text>{_("Fedora is successfully installed. To begin using it, reboot your system.")}</Text>
-                  }
-                  secondary={
-                      status === "success" &&
-                      <Button onClick={exitGui}>{_("Reboot")}</Button>
-                  }
-                  title={title}
-                />
-            </Bullseye>
+            <EmptyStatePanel
+              icon={icon}
+              loading={!icon}
+              paragraph={
```

Comment:
Since the paragraph content is SO long, you can store it in a variable and then use:

`paragraph={installationProgressStepper}`

Reply:
Thanks for the suggestion but I think I'd like to keep the amount of indirection low in the rendering section, at least for now. :)